### PR TITLE
man page: add note about issue with SELinux

### PR DESCRIPTION
**Pull request by zypA13510**
Ref: #5538

### CODE REVIEW

**Comment by giuseppe** [comment]
please sign your commit with `git commit --amend -s` and `push -f` it

File: docs/source/markdown/podman-generate-systemd.1.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ Note that this command is not supported for the remote client.
 
 Generate files instead of printing to stdout.  The generated files are named {container,pod}-{ID,name}.service and will be placed in the current working directory.
 
+Note: On a system with SELinux enabled, the generated files will inherit contexts from current working directory. Depending on your SELinux setup, you may need to change it using `restorecon`, `chcon`, or `semanage`, to make them work with systemd.
```
Comment (vrothberg) [inline docs/source/markdown/podman-generate-systemd.1.md:21]:
```suggestion
Note: On a system with SELinux enabled, the generated files will inherit contexts from the current working directory. Depending on your SELinux setup, you may need to change it using `restorecon`, `chcon`, or `semanage`, to make them work with systemd.
```

Comment (vrothberg) [reply docs/source/markdown/podman-generate-systemd.1.md:21]: I would also mention that usage of `mv -Z` and `cp -Z` which will take care of that implicitly.

**Comment by zypA13510** [comment]
Signed, I hope GitHub privacy email is not a problem, is it?

**Comment by mheon** [comment]
Nice. LGTM
/approve
/ok-to-test

File: docs/source/markdown/podman-generate-systemd.1.md [hunk 2]
```diff
@@ -18,6 +18,8 @@ Note that this command is not supported for the remote client.
 
 Generate files instead of printing to stdout.  The generated files are named {container,pod}-{ID,name}.service and will be placed in the current working directory.
 
+Note: On a system with SELinux enabled, the generated files will inherit contexts from the current working directory. Depending on your SELinux setup, you may need to change it using `restorecon`, `chcon`, or `semanage`, to make them work with systemd. Alternatively, use `-Z` switch when running `mv` or `cp`.
+
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-generate-systemd.1.md:22]: I'm not a fan of the words 'you' or 'your' in a man page.  Could we instead say:
```
Note: On a system with SELinux enabled, the generated files will inherit contexts from the current working directory. Depending on the  SELinux setup, changes to the generated files using `restorecon`, `chcon`, or `semanage` may be required to allow systemd to work on the system. Alternatively, use the `-Z` option when running `mv` or `cp`.
```

Comment (TomSweeneyRedHat) [reply docs/source/markdown/podman-generate-systemd.1.md:22]: missing 'the' in front of "`-Z` switch"

Comment (zypA13510) [reply docs/source/markdown/podman-generate-systemd.1.md:22]: > I'm not a fan of the words 'you' or 'your' in a man page.

There are precedents, just [a search for "note" in docs](https://github.com/search?q=Note+repo%3Acontainers%2Flibpod+path%3A%2Fdocs%2Fsource%2Fmarkdown&type=Code&ref=advsearch&l=&l=) reveals this:

> Note: You can also override the default path of the authentication file by setting the **REGISTRY_AUTH_FILE**

But of course, if you insist.

Comment (rhatdan) [reply docs/source/markdown/podman-generate-systemd.1.md:22]: Yes please make the change, and if you could, please change the REGISTRY_AUTH_FILE one also,

Comment (zypA13510) [reply docs/source/markdown/podman-generate-systemd.1.md:22]: Done. I made a slight change:
```diff
- ... may be required to allow systemd to work on the system.
+ ... may be required to allow systemd to access these files.
```

I hope it's ok?

**Pull request merged**